proto/encore/parser/schema/v1/schema.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v3.19.3
+// 	protoc        v3.14.0
```

Comment:
You _might_ want to update your Protoc to the newest version, otherwise we're going to constantly be changing these lines each time we change/regenerate the proto files.